runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/tasks/DokkaCollectorTask.kt

```diff
@@ -15,15 +15,15 @@ abstract class DokkaCollectorTask : AbstractDokkaParentTask() {
 
     override fun buildDokkaConfiguration(): DokkaConfigurationImpl {
         val initialDokkaConfiguration = DokkaConfigurationImpl(
-            moduleName = moduleName.getSafe(),
-            outputDir = outputDirectory.getSafe(),
-            cacheRoot = cacheRoot.getSafe(),
-            failOnWarning = failOnWarning.getSafe(),
-            offlineMode = offlineMode.getSafe(),
+            moduleName = moduleName.get(),
+            outputDir = outputDirectory.get(),
+            cacheRoot = cacheRoot.get(),
```

Comment:
Looks like this needs to be `.orNull`, there's a failing test `Collector0IntegrationTest`:

```
Caused by: org.gradle.api.internal.provider.MissingValueException: Cannot query the value of task ':moduleA:dokkaHtmlCollector' property 'cacheRoot' because it has no value available.
  at org.gradle.api.internal.provider.AbstractMinimalProvider.get(AbstractMinimalProvider.java:86)
  at org.jetbrains.dokka.gradle.DokkaCollectorTask.buildDokkaConfiguration$gradle_plugin(DokkaCollectorTask.kt:20)
```

The rest looks good to me :+1:

Reply:
```suggestion
            cacheRoot = cacheRoot.orNull,
```

Good catch!

Reply:
It would be nice to have a local test for this - I'll see if I can create one.